src/components/EditTodoForm.js

```diff
@@ -0,0 +1,21 @@
+import Reactת, {useState} from 'react'
+
+export const EditTodoForm = ({editTodo, task}) => {
+    const [value, setValue]=useState(task.task)
```

Comment:
its not recommended to init a state from a prop, you can just change the state directly